Dockerfile

```diff
@@ -1,24 +1,35 @@
-FROM registry.opensource.zalan.do/stups/python:3.5.1-23
+FROM ubuntu:16.04
```

Comment:
Because it is:
* smaller
* has python3-boto3 deb package (we don't need to use pip)
* we are not using anything special provided by stups/python
* I don't want to track latest version of stups/python